MediaBrowser.Providers/Subtitles/SubtitleManager.cs

```diff
@@ -251,7 +249,7 @@ private async Task TrySaveToFiles(Stream stream, List<string> savePaths)
                     Directory.CreateDirectory(Path.GetDirectoryName(savePath) ?? throw new InvalidOperationException("Path can't be a root directory."));
 
                     var fileOptions = AsyncFile.WriteOptions;
-                    fileOptions.Mode = FileMode.CreateNew;
+                    fileOptions.Mode = FileMode.Create;
```

Comment:
I'm not very experienced with C#, but according to [the docs](https://learn.microsoft.com/en-us/dotnet/api/system.io.filemode?view=net-8.0#fields), `FileMode.Create` means "If the file already exists, it will be overwritten". 
Is this what we want? 

If I remember correctly, a long time ago downloading multiple subtitles of the same language would create multiple files, in the format of `<subtitle-name>.<lang>.<number>.srt` (e.g. MovieTitle.en.1.srt). I liked this solution as it meant it was easy to test multiple subtitles of the same language without exiting out of the player to download another subtitle. After testing, I can then simply only keep the one that works best.

Reply:
I just read the PR title and description again and realize now that you already mention this :laughing: 

I would personally prefer the alternative approach of appending a number if the file already exists.